tests/unit/query_graph/test_extract_operation_structure.py

```diff
@@ -512,7 +512,9 @@ def test_extract_operation__join_double_aggregations(
     order_size_column = {
         "name": "ord_size",
         "columns": [order_id_source_data],
-        "transforms": ["item_groupby", "add(value=123)"],
+        "transforms": [
+            "join_feature(feature_entity_column='order_id', name='ord_size', view_entity_column='order_id')"
+        ],
```

Comment:
@YS-L One of the side effect of this changes is the `transforms` attribute of the operation structure.